rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/MatcherMonadSpec.scala

```diff
@@ -66,10 +66,9 @@ class MatcherMonadSpec extends FlatSpec {
       _        <- cost.set(Cost(0))
       a        = attemptOpt(modifyStates)
       res      <- runFirstWithCost(a)
-      (_, r)   = res
       phloLeft <- cost.get
       _        = assert(phloLeft.value == 1)
-    } yield (assert(r == Some((Map(42 -> Par()), Some(())))))).unsafeRunSync
+    } yield (assert(res == Some((Map(42 -> Par()), Some(())))))).unsafeRunSync
```

Comment:
having the `runWithCost(f, phlo)` wrapper I mentioned earlier would make those changes unneeded :)